FluentUI.Demo/src/main/res/layout/activity_app_bar_layout.xml

```diff
@@ -9,4 +9,6 @@
     android:id="@+id/app_bar_layout_list"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:importantForAccessibility="no"
+    android:background="?attr/fluentuiBackgroundColor"
```

Comment:
I think it was decided not to use attr directly in demo after modularization ?

Reply:
It is used at many places in the demo app as the base theme is Fluent Theme.

As per my understanding the discussion was based on the fact that Demo app theme should not extend FluentTheme 

These all should be changed once, we switch to another theme at once, Or i can raise a PR changing this for now. But it still will be used in other places in Demo app

Reply:
Let's not introduce  one more attr to be replaced , a New color should work fine